### PR TITLE
Urdf rs

### PR DESCRIPTION
**Pull request by Vrixyz**
- Builds on top of https://github.com/dimforge/rapier/pull/744

See commit c7a571041028c22d40feef72d75eca946bfc7908

I tested example urdf.

I added support for capsules, but lack the knowledge to find relevant files.

### CODE REVIEW

File: crates/rapier3d-urdf/CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
+
+### Added
+
+- Add optional support for Collada and Wavefront files through new feature flags `collada` and `wavefront`.
+- Add support for capsule urdf geometry
+=======
 ### Added
 
 - Add optional support for Collada and Wavefront files through new feature flags `collada` and `wavefront`.
+>>>>>>> @{-1}
```
Comment (sebcrozet) [inline crates/rapier3d-urdf/CHANGELOG.md:17]: Looks like some unresolved merge conflict.

File: crates/rapier3d-meshloader/src/lib.rs [hunk 2]
```diff
@@ -48,12 +48,14 @@ pub fn load_from_path(
...
-            raw_mesh,
-        }));
+        colliders.push(match shape {
+            Ok((shape, pose)) => Ok(LoadedShape {
+                shape,
+                pose,
+                raw_mesh,
+            }),
+            Err(e) => Err(e),
+        });
```
Comment (sebcrozet) [inline crates/rapier3d-meshloader/src/lib.rs:58]: Why was that change needed?

**Review by sebcrozet — APPROVED** [review]
Just a couple of remarks but looks good overall. Thanks!

File: crates/rapier3d-meshloader/src/lib.rs (hunk 2 above)
Comment (Vrixyz) [reply crates/rapier3d-meshloader/src/lib.rs:58]: I probably added a log in between and then forgot, I'll revert it
